common/src/main/scala/explore/utils/package.scala

```diff
@@ -72,6 +72,13 @@ package object utils {
         pot.fold(pendingRender, errorRender, valueRender)
     )
 
+  def potRenderBasic[A](
```

Comment:
@rpiaggio WDYT, a potRender for dummies (or just me if you don't want to make it plural)

Reply:
Sounds good if we are going to attempt moving to a world with less reuse.

Maybe this can be `potRender` and we can rename the other one to `potRenderWithReuse`.